users/models.py

```diff
@@ -62,6 +68,9 @@ def create_superuser(self, email: str, password: str) -> "CustomUser":
             email, password, is_staff=True, is_superuser=True, user_type=CustomUser.UserType.ADMIN.name
         )
 
+    def get_queryset(self) -> CustomUserQuerySet:
```

Comment:
Don't do this, it will then basically add prefetching reports to **every** call for users. We don't want that. Just make `reports_connected_to_user` a method of `CustomUserManager`.

Reply:
Sure, we had discussion about it.